apps/strapi-cms/config/middlewares.ts

```diff
@@ -0,0 +1,12 @@
+export default [
+  'strapi::errors',
+  'strapi::security',
+  'strapi::cors',
+  'strapi::poweredBy',
+  'strapi::logger',
+  'strapi::query',
+  'strapi::body',
+  'strapi::session',
+  'strapi::favicon',
+  'strapi::public',
+];
```

Comment:
@datalek sorry i had delete your comment by mistake :pray: 
About this middlewares here some additional detail.
These are the middleware installed by default by strapi. Only `logger`, `poweredBy` and `session` are not required. I have removed all optionals middlewares except for `logger` that can be usefull for debugging, however i think also `session` could be useful if we will use strapi to mange auth. [Here](https://docs.strapi.io/dev-docs/configurations/middlewares) you can find some more info in depth.

Reply:
Don't worry ^^. 👍